src/main/java/org/spongepowered/api/util/command/CommandSource.java

```diff
@@ -44,6 +46,17 @@
     String getName();
 
     /**
+     * Gets the locale used by this command source. If this
+     * {@link CommandSource} does have a {@link Locale} configured or does not
+     * support configuring a {@link Locale}, the system default is used.
+     *
+     * @return The locale used by this command source
+     */
+    default Locale getLocale() {
```

Comment:
I'm not sure about this one. The alternative would be adding a `Localed` interface.
In the end I decided to use this version because in command handling you are usually pushing CommandSenders around and would like to save the plugin authors to check for `Localed` (or `Player` currently) to get the right language for that thing. And since this is a default method this won't cost us anything.

The `CommandBlock` will use most likely the same language as the `Console` does. But maybe (in the future when the console language has been decoupled from Locale.getDefault()) the console uses `Chinese` (configured somewhere) because that way the mods can better understand messages send by plugins, but Exceptions should still be printed in English (system locale) so the admin can google them and will get proper answers and can even use them to post them here/somewhere.

In addition to that I could image that some Remote console implementations would want to be able to change their locale too.

````java
CommandSender sender = ...
Locale locale = sender.getLocale();
````
vs
````java
CommandSender sender = ...
Locale locale;
if (sender instanceOf Player) {
    locale = sender.getLocale();
} else if (sender instanceOf MyMultilanguageConsole) { // Will be forgotten most of the times
    locale = sender.getLocale();
} else {
    locale = Locale.getDefault(); // Is this really the servers default?
}
````